src/components/Navbar.tsx

```diff
@@ -100,6 +100,10 @@ const MobileBar = styled.div`
 
     @media (max-width: 780px) {
       display: block;
+      outline: none;
```

Comment:
Is this essential for eslint/some a11y rules to not throw errors? Maybe we can open another PR for style changes?

Reply:
it's not essential for es-lint, but - it's connected with the need of changing 'span' to 'button' (because this is a clickable element) - so I needed to add some changes in style

Reply:
http://www.outlinenone.com/

Reply:
Yeah, right. Thanks @a-dabrowski. @MariaRukat if we don't have styles that indicates focus status somehow maybe @karoli-na can help :)

Reply:
You are right, I forgot about it. So now I will leave outline and we can discuss it later with @karoli-na if we want to make our own styles for outline.